docs/pages/guides/meshery-metrics.md

```diff
@@ -1,99 +1,195 @@
 ---
 layout: default
 title: Using Metrics in Meshery
-description: How to connected and use Prometheus and*Grafana*metrics in Meshery
+description: How to connect and use Prometheus and Grafana metrics in Meshery
 permalink: guides/meshery-metrics
 type: Guides
 ---
 
-In order to generate performance test reports of service meshes and their workloads, Meshery uses **Grafana** and/or **Prometheus** as visualization and metrics systems, respectively. This document outlines the requirements necessary for Meshery to connect to these systems. Steps vary depending upon the service mesh and its configuration.
+# Connect and use metrics in Meshery
 
-* TOC
-{:toc}
+Meshery provides performance reports, including performance test results, node resource metrics etc. so that operators may easily understand the overhead of their service mesh's control plane and data plane in context of the overhead incurred on nodes running within the cluster. In order to generate performance test reports of service meshes and their workloads, Meshery uses `Grafana` and/or `Prometheus` as visualization and metrics systems, respectively. This guide outlines the requirements necessary for Meshery to connect to these systems. The steps may vary depending upon the service mesh and its configuration.
 
+In order to pull in these environment metrics, you can also manually configure Meshery to connect with your existing Grafana and/or Prometheus instances through the Meshery dashboard. Once they have been loaded and are displayed on the screen, you may also perform an *ad-hoc* test to check Meshery's connection status.
 
-## Istio + Minikube
+## Tutorial Guide
 
-In this scenario, Meshery has deployed Istio to a Kubernetes cluster running in Minikube. The service ports of*Grafana*and Prometheus need to be exposed in order for Meshery to connect to and interact with these visualization and metrics systems.
+Follow along with this tutorial to set up and integrate Grafana and Prometheus instances on top of your service mesh and run performance tests:
 
-### Get minikube ip
+* Deploy a service mesh and any available sample application
```

Comment:
Will you consider defining a clearer separation between generic and example-specific content?